spec/spec_helper.rb

```diff
@@ -4,4 +4,5 @@
 
 $TESTING=true
 $:.unshift File.join(File.dirname(__FILE__), '..', 'lib')
+require 'timeout'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.